apps/sms/views/conversation/js/conversation.js

```diff
@@ -594,9 +612,7 @@ var ConversationView = {
 
   afterEnterComposer: function conv_afterEnterComposer(args) {
     // TODO Bug 1010223: should move to beforeEnter
-    if (args.activity) {
-      this.handleActivity(args.activity);
```

Comment:
\0/ Hope it works for all cases we have! I don't really care that now we always have "Replace existing draft" when going back, as I'm going to get rid of this dialog very soon :)